widgets/src/google/Common2D.js

```diff
@@ -17,11 +17,28 @@
 
     Common2D.prototype.publish("paletteID", "default", "set", "Palette ID", Common2D.prototype._palette.switch());
 
+    Common2D.prototype.getChartOptions = function () {
+        var chartOptions = Common.prototype.getChartOptions.call(this);
+        chartOptions.series = initSeries(this.getNumSeries());
```

Comment:
it could but this is a placeholder for now to start building upon some of the more complex features of google charts. I expect this to change.